.kokoro/release/stage.sh

```diff
@@ -35,11 +35,15 @@ mvn clean deploy -B \
   -Dgpg.homedir=${GPG_HOMEDIR} \
   -P release
 
-# During shadow-mode, do not release.
-#if [[ -n "${AUTORELEASE_PR}" ]]
-#then
-#  mvn nexus-staging:release -B \
-#    -DperformRelease=true \
-#    --settings=settings.xml
-#fi
+# The job triggered by Release Please (release-trigger) has this AUTORELEASE_PR
+# environment variable. Fusion also lets us to specify this variable.
+if [[ -n "${AUTORELEASE_PR}" ]]
```

Comment:
If needed, we can also release manually after it's staged. It would allow us to do a final check.

Reply:
As talked, I'll get your manual help in case it doesn't work.